src/conductor.ts

```diff
@@ -156,10 +156,37 @@ export class Conductor {
       }
       source = await this._backend.downloadDnaRemote((source as T.DnaUrl).url)
     }
-    const registerDnaReq: RegisterDnaRequest = { source, uuid, properties }
+    const registerDnaReq: RegisterDnaRequest = { ...source, uuid, properties }
     return await this.adminClient!.registerDna(registerDnaReq)
   }
 
+  // this function will install an app bundle as generated by hc app pack
+  installBundledHapp = async (bundleSource: AppBundleSource, agentPubKey?: AgentPubKey): Promise<T.InstalledHapp> => {
```

Comment:
```suggestion
  installBundledHapp = async (bundleSource: AppBundleSource, agentPubKey?: AgentPubKey, installedAppId?: string): Promise<T.InstalledHapp> => {
```

Reply:
These conductor changes align with the proposed change to the players `installBundledHapp` method.